<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提取PDF中的表格</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tool.ico</normaloff>tool.ico</iconset>
  </property>
  <widget class="QLabel" name="label_CSVExt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>241</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>.csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Extract">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>321</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提取表格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CSVPos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>181</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>保存位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_PdfPos">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>51</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>请选择待提取PDF中的位置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_OpenPdfPos">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>51</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PdfPos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>52</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PDF 位置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_OpenFile">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>321</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开保存的CSV文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_OpenCSVPos">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_CustomPages">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自定义页数</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_AllPages">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全部</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_ExtractPages">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>待提取页数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CSVName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>241</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>保存文件名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_CSVPos">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>请选择提取结果输出位置，默认为桌面</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_CustomPages">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>输入待提取表格所在页数，若有多个数字以“，”分隔开</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_CSVName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>241</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>默认为Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
